transcriptic/cli.py

```diff
@@ -179,6 +179,68 @@ def cli(ctx, api_root, email, token, organization, config):
             pass
 
 
+@cli.command("exec")
+@click.argument("autoprotocol", type=click.File("r"), default=sys.stdin)
+@click.option(
+    "--api", "-a", help="The api endpoint of your scle test workcell instance."
+)
+@click.option(
+    "--workcellId",
+    "-wc",
+    help="The workcell id to use for the device set. This is not permitted along with the `deviceSet` option.",
+)
+@click.option(
+    "--deviceSet",
+    "-d",
+    type=click.File("r"),
+    help="A DeviceSet json file to use for scheduling. This is not permitted along with the `workcellId` option.",
+)
+@click.option(
+    "--timeLimit",
+    "-t",
+    type=click.INT,
+    default=30,
+    help="The maximum time in seconds to spend scheduling. The scheduler will use all the time until an optimal solution is found.",
+)
+@click.option(
+    "--partitionGroupSize",
+    type=click.INT,
+    default=None,
+    help="The number of x_partition groups to be scheduled together.",
```

Comment:
yea... for these.. I guess its okay for now, but it really wouldn't make much sense to most users